carbonmark/components/pages/Project/BuyOptions/SellerListing.tsx

```diff
@@ -88,33 +93,24 @@ export const SellerListing: FC<Props> = (props) => {
         </Text>
       </div>
 
-      {!address && !isConnected && (
+      <div className={styles.buttons}>
         <ButtonPrimary
-          className={styles.buyButton}
-          label={t({
-            id: "shared.connect_to_buy",
-            message: "Sign In / Connect To Buy",
-          })}
-          onClick={toggleModal}
+          label={t`Buy`}
+          renderLink={(linkProps) => <Link {...linkProps} />}
+          onClick={() => {
+            LO.track("Purchase - Listing: Buy Clicked");
```

Comment:
Not sure about these analytics events here. I just introduced a new event I guess. It's based on the pool equivalent